libs/agno/agno/team/team.py

```diff
@@ -107,6 +107,9 @@ class Team:
     session_name: Optional[str] = None
     # Session state (stored in the database to persist across runs)
     session_state: Optional[Dict[str, Any]] = None
+
+    # Team session state (stored in the database to persist across team runs)
```

Comment:
```suggestion
    # Team session state (shared between team leaders and team members)
```
It doesn't require a DB to exist

Reply:
done